bindings/stronghold-nodejs/README.md

```diff
@@ -0,0 +1,44 @@
+# IOTA Identity Stronghold Node.js
+
+> This is the beta version of the official Stronghold Node.js plugin for [IOTA Identity Wasm](../wasm).
```

Comment:
Another relative link that should be replaced.

I also think we should specify this is specifically for `Account Storage`.

```suggestion
# IOTA Identity Stronghold Node.js

This is the beta version of the official Stronghold Account Storage plugin for Node.js with [IOTA Identity Wasm](https://github.com/iotaledger/identity.rs/tree/main/bindings/wasm).
```